utils/src/main/scala/za/co/absa/hermes/utils/SparkTestBase.scala

```diff
@@ -16,12 +16,19 @@
 package za.co.absa.hermes.utils
 
 import com.typesafe.config.{Config, ConfigFactory}
+import org.apache.log4j.{Level, Logger}
 import org.apache.spark.sql.SparkSession
 
 trait SparkTestBase { self =>
   val config: Config = ConfigFactory.load()
   val sparkMaster: String = config.getString("utils.sparkTestBaseMaster")
 
+  System.setProperty("user.timezone", "UTC")
+
+  // Do not display INFO entries for tests
+  Logger.getLogger("org").setLevel(Level.WARN)
```

Comment:
I saw in several places that you use log level WARN in tests. It might be a matter of taste, but I turn off logs as much as possible in tests so that for all passing tests there should be no log entries.